README.md

```diff
@@ -24,6 +24,37 @@ In order to run a different RDBMS, the environment variable ``DB_VENDOR`` can be
 
 Uninstalling Janeway is as simple as running ``make uninstall`` which will delete all related containers as well as wipe the database volume.
 
+# Using Lando for an alternative development environment (optional)
```

Comment:
This block of text is _very_ long. Can I suggest it gets moved to docs? We will then add similar entries for Docker and Manual and link to them all from a single section in the readme.

Reply:
Makes sense, would you like to include the docs refactoring as part of this work?

Reply:
moved to docs